components/List/examples/MDCSelfSizingStereoCellExample.m

```diff
@@ -40,7 +40,7 @@ - (void)viewDidLoad {
   [super viewDidLoad];
   self.parentViewController.automaticallyAdjustsScrollViewInsets = NO;
   self.automaticallyAdjustsScrollViewInsets = NO;
-  self.listScheme = [[MDCListScheme alloc] init];
+  self.containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
Nit: Please check if it's not `nil` before assigning a value. The catalogs should be injecting a value to each example.

Reply:
done